browser/src/GenePage/StructuralVariantsInGene.tsx

```diff
@@ -29,6 +29,7 @@ const StructuralVariantsInGene = ({ datasetId, gene, zoomRegion, ...rest }: Prop
       gene(gene_id: $geneId, reference_genome: $referenceGenome) {
         structural_variants(dataset: $datasetId) {
           ac
+	  ac_hemi
```

Comment:
Nitpick: I think the formatter doesn't run due to our noted strange practicing of having a big ol' string that represents the query -- I'd prefer consistent indentation here but fully realize this is nitpicky and this is non-blocking

Reply:
@rileyhgrant this looks like it's aligned correctly now--I did see, earlier, some odd discrepancies between the diffs and my editor. not sure what to make of all this but I will clean it up if there is anything to clean up.